session/payment/session_payments.go

```diff
@@ -45,22 +47,26 @@ type SessionPayments struct {
 	peerPromiseSender PeerPromiseSender
 	promiseTracker    PromiseTracker
 	balanceTracker    BalanceTracker
+	sendRetryDuration time.Duration
 }
 
 // NewSessionPayments returns a new instance of consumer payment orchestrator
-func NewSessionPayments(balanceChan chan balance.Message, peerPromiseSender PeerPromiseSender, promiseTracker PromiseTracker, balanceTracker BalanceTracker) *SessionPayments {
+func NewSessionPayments(balanceChan chan balance.Message, peerPromiseSender PeerPromiseSender, promiseTracker PromiseTracker, balanceTracker BalanceTracker, sendRetryDuration time.Duration) *SessionPayments {
 	return &SessionPayments{
 		stop:              make(chan struct{}),
 		balanceChan:       balanceChan,
 		peerPromiseSender: peerPromiseSender,
 		promiseTracker:    promiseTracker,
 		balanceTracker:    balanceTracker,
+		sendRetryDuration: sendRetryDuration,
 	}
 }
 
 // balanceDifferenceThreshold determines the threshold where we'll cancel the session if there's a missmatch larger than the threshold provided between the provider and the consumer balances
 const balanceDifferenceThreshold uint64 = 20
```

Comment:
I don't see it being used anywhere. If balance miss-match would occur, we should print out bald warning. 
In our case when we would pay for time, this should not occur.

If we would make payment by data transferred, that would mean we would have increasing balance miss-match. In such a case static threshold would not make sense, since difference would continuously increase. I would propose % difference,  not absolute.

Reply:
#812